### PR TITLE
Update developer documentation + replace gitlab.inria.fr by github.com

### PR DESCRIPTION
**Pull request by srcansiz**
**IMPORTANT**: Please merge after #767 is merged

### CODE REVIEW

File: docs/developer/usage_and_tools.md [hunk 1]
```diff
@@ -76,7 +95,7 @@ Current roles in Fed-BioMed development process are:
     * Currently, Core Developers also give final approval and merge the merge requests
     * new Core Developers are chosen by the existing Core Developers among the Contributors.
 
-In terms of mapping to accounts and roles on Gitlab server:
+In terms of mapping to accounts and roles on GitHub Fed-BioMed organization:
```
Comment (mvesin) [inline docs/developer/usage_and_tools.md:98]: We need the `Contributors` in two categories:
- `Contributors` that dont need to belong to the fedbiomed repository or organization, and submit pull requests from forks.
- `Team Developers` that are members of the repository with `write` access and usually work on feature branches
without forking the repository. They are regular contributors working in coordinated manner with other team developers. They are chosen by existing team developers from contributors.
Will rewrite CONTRIBUTING.md and usage_and_tools.md for that

Comment (mvesin) [reply docs/developer/usage_and_tools.md]: We need to adapt the label process to:
- introduce `needs-triage` & differentiate user/developer submitted issue
- differentiate the roles of team developers / contributors

**Review by mvesin — APPROVED** [review]

File: docs_src/fedbiomed_dev_usage_issues.odg
Comment (srcansiz) [inline docs_src/fedbiomed_dev_usage_issues.odg:1]: Hi @mvesin, 

I didn't where these `.odg` files are used.  However, if they are used in `docs` it is better to put in `docs/assets/img`. Also, the size of `odg` format is not optimized. Could you please convert them to `jpg`.

Comment (mvesin) [reply docs_src/fedbiomed_dev_usage_issues.odg:1]: Thanks @srcansiz 

The `.odg` files are the source file for the schemas (OpenDocument graphics).
On gitlab they were kept in `$FEDBIOMED_DOC_DIR/src`
They are needed to modify the images.
Where should we keep them, then ?

We cannot compress as this is the (standard) OD graphics file.

Comment (mvesin) [reply docs_src/fedbiomed_dev_usage_issues.odg:1]: hi @srcansiz 
> Where should we keep them, then ?

Then do you think we should remove the `docs_src` directory (and not keep sources of images in the repository) ? Or rename it ?

Thx

**Review by mvesin — APPROVED** [review]

**Pull request merged**